src/feature/dircache/dircache.c

```diff
@@ -1449,13 +1449,19 @@ handle_get_next_bandwidth(dir_connection_t *conn,
   (void)args;
```

Comment:
args are used, so this line is not needed now.

Reply:
last commit should fix that